fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorContext.java

```diff
@@ -237,6 +241,10 @@ public TablePath getTablePathById(long tableId) {
         return this.tablePathById.get(tableId);
     }
 
+    public Long getTableIdByPath(TablePath tablePath) {
```

Comment:
not used, can be removed.

Reply:
This will be used in the upcoming PR, so I hope can be kept.